sdk/videoanalyzer/azure-media-videoanalyzer-edge/src/samples/java/com/azure/media/videoanalyzer/edge/LvaInvokeModuleSample.java

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.media.videoanalyzer.edge;
+
+import com.azure.media.videoanalyzer.edge.models.*;
+
+import com.azure.media.videoanalyzer.edge.models.*;
+import com.microsoft.azure.sdk.iot.service.devicetwin.DeviceMethod;
+import com.microsoft.azure.sdk.iot.service.devicetwin.MethodResult;
+import java.io.IOException;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
+
+import java.util.Arrays;
+
+public class LvaInvokeModuleSample {
```

Comment:
Add javadoc to the class and all public methods to describe what this sample is for.